### DISCUSSION TITLE
Not working on Windows initial startup

### DISCUSSION THREAD

**Discussion by drspamilton** [JPersson77/LGTVCompanion#184] [General]
Thanks for your work on this project. I have been using it for over a year now. I don't think this issue has to do with LGTVCompanion, but did something change recently with Windows that you know of? It no longer seems to be initiating the monitor power saving on the startup lock screen. It used to function the same (turns off after a minute) at startup as it did any other time. Now, I have to log in then lock it to get it to work. Any ideas? Thanks.

**Comment by JPersson77** [comment]
Hi, I've not noticed any other reports mentioning a changed behaviour with windows "turn off display after X minutes" at the login screen. Please check if the log is indicating any errors. 

I will try to replicate

**Reply by drspamilton** [reply]
I think I've realized that this only happens when booting after "update and shutdown". For some reason, it doesn't boot to the normal login screen after a windows update and doesn't allow the required network connectivity. Thanks for the response and work.
